docs/system/wires_fuses.md

```diff
@@ -2,15 +2,84 @@
 
 ## Wires
 
-::: warning TODO
-- Suitable thickness for different applications
-- Voltage drop depending on length and current (possibly interactive diagram)
-:::
+For any current carrying conductor, resistance ($R$) of the conductor is given by
+
+$$R = \frac{\rho * L}{A}$$
+
+Hence the efficiency of current carrying conductor depends mainly on length ($L$), cross-sectional area ($A$) and specific resistance ($\rho$) of the material. In addition, parameters like ambient temperature and installation conditions play an important role in maximum current carrying capacity of the wire. Maximum current carrying capacity ensures that neither the metal(wire) nor the insulation around it reaches its melting point.
+
+### 1. Cross section area
+
+Higher the cross-sectional area of the conductor, there will be more area for the electrons to flow and hence lesser resistance offered to the current flow. So its current carrying capacity and efficiency will be more. Also, less heat will be dissipated in case of higher cross-sectional area.
+
+### 2. Length
+
+Higher the length of the conductor, greater is the effective resistance offered by the path and hence more power will be dissipated. This limits the maximum current carrying capacity of the conductor and reduces its efficiency as well.
+
+### 3. Specific resistance
+
+Specific resistance is the property of the material used for the wire. Different materials like copper, aluminium, silver and others exhibit different value of specific resistance. Materials like silver with low specific resistance are more conductive in nature, but are more expensive as well. Hence wire of suitable material should be used according to the requirement. The following table gives the value of specific resistance of few materials.
+
+| Material          | Specific Resistance $\rho$ ($\Omega$ mm2/m)  |
+|:-----------------:|:--------------------------------------------:|
+|   Silver          |                  0.0159                      |
+|   Copper          |                  0.0168                      |
+| Copper,annealed   |                  0.0172                      |
+|   Iron            |                  0.0971                      |
+
+### 4. Ambient temperature
+
+Higher the ambient temperature, less heat (generated by current flowing through wire) will be sufficient for the insulation to melt. Hence ambient temperature has to be kept low to achieve higher current carrying capacity of the wire. The deviation of current carrying capacity (from 30C) depending on ambient temperature is given in the below table, which is according to the VDE 0298 norm.
+
+| Ambient Temperature (C)   | Deviating Factor   |
```

Comment:
Also degree C.

And is it a derating factor? Deviating factor sounds strange to me. Or may deviation factor?